src/Build/BuildScripts/Module.build

```diff
@@ -26,4 +29,10 @@
     <Copy SourceFiles="@(PersonaBar-scripts)" DestinationFolder="$(ModuleFolderName)\Scripts" />
     <Copy SourceFiles="@(PersonaBar-css)" DestinationFolder="$(ModuleFolderName)\Css" />
   </Target>
+  <Target Name="RunYarn" Condition="$(YarnWorkingDirectory.Length) > 0">
+    <Message Text="Yarn Working Directory: $(MSBuildProjectDirectory)\$(YarnWorkingDirectory)\"  Importance="high" />
+    <Yarn Command="install" WorkingDirectory="$(MSBuildProjectDirectory)\$(YarnWorkingDirectory)\" IgnoreExitCode="true"  />
+    <Yarn Command="run build" WorkingDirectory="$(MSBuildProjectDirectory)\$(YarnWorkingDirectory)\" IgnoreExitCode="true"  />
+	<Message Text="Yarn Finished."  Importance="high" />
```

Comment:
Do we need/want these messages all of the time, or is this just for debugging?

Reply:
It was just for debugging, I'm happy to remove them if you would rather not have them show up in the build log.

Reply:
Fixed in both pull requests.